prov/verbs/src/verbs_rma.c

```diff
@@ -90,6 +90,9 @@ fi_ibv_msg_ep_rma_writemsg(struct fid_ep *ep_fid, const struct fi_msg_rma *msg,
 		.wr.rdma.rkey = (uint32_t)msg->rma_iov->key,
 	};
 
+	if (msg->rma_iov_count > ep->info->tx_attr->rma_iov_limit)
+		return -FI_EINVAL;
```

Comment:
The rma iov limits are reported as part of the fi_info attributes.  The application is expected to adhere to these limits.  We don't want validation checks in the fast path, as these indicate that the application has been coded incorrectly and is not a run time error.  At best these should be asserts.

Reply:
Thanks for the comment. I understand your argument and I will remove the checks from the fast path. I'm closing the PR.